README.md

```diff
@@ -118,12 +118,9 @@ kubectl apply -k https://raw.githubusercontent.com/OpenZWave/Zwave2Mqtt/master/k
 
 ### NodeJS or PKG version
 
-1. Firstly you need to install [Open-Zwave](https://github.com/OpenZWave/open-zwave) library on your system.
-
-   If you are using Ubuntu:
```

Comment:
Why did you delete this?

Reply:
I didn't delete it, I just added the world "the" before "Open-ZWave library" so the line has been replaced with a new one.